ush/config_defaults.yaml

```diff
@@ -2450,9 +2450,9 @@ verification:
   #
   # File name and path templates used in the verification tasks.
   #
-  FCST_SUBDIR_TEMPLATE: '{% if user.RUN_ENVIR == "nco" %}${NET}.{init?fmt=%Y%m%d?shift=-${time_lag}}/{init?fmt=%H?shift=-${time_lag}}{% else %}{init?fmt=%Y%m%d%H?shift=-${time_lag}}{% if global.DO_ENSEMBLE %}/${ensmem_name}{% endif %}/postprd{% endif %}'
-  FCST_FN_TEMPLATE: '${NET}.t{init?fmt=%H?shift=-${time_lag}}z{% if user.RUN_ENVIR == "nco" and global.DO_ENSEMBLE %}.${ensmem_name}{% endif %}.prslev.f{lead?fmt=%HHH?shift=${time_lag}}.${POST_OUTPUT_DOMAIN_NAME}.grib2'
-  FCST_FN_METPROC_TEMPLATE: '${NET}.t{init?fmt=%H}z{% if user.RUN_ENVIR == "nco" and global.DO_ENSEMBLE %}.${ensmem_name}{% endif %}.prslev.f{lead?fmt=%HHH}.${POST_OUTPUT_DOMAIN_NAME}_a${ACCUM_HH}h.nc'
+  FCST_SUBDIR_TEMPLATE: '{% if user.RUN_ENVIR == "nco" %}${NET_default}.{init?fmt=%Y%m%d?shift=-${time_lag}}/{init?fmt=%H?shift=-${time_lag}}{% else %}{init?fmt=%Y%m%d%H?shift=-${time_lag}}{% if global.DO_ENSEMBLE %}/${ensmem_name}{% endif %}/postprd{% endif %}'
+  FCST_FN_TEMPLATE: '${NET_default}.t{init?fmt=%H?shift=-${time_lag}}z{% if user.RUN_ENVIR == "nco" and global.DO_ENSEMBLE %}.${ensmem_name}{% endif %}.prslev.f{lead?fmt=%HHH?shift=${time_lag}}.${POST_OUTPUT_DOMAIN_NAME}.grib2'
+  FCST_FN_METPROC_TEMPLATE: '${NET_default}.t{init?fmt=%H}z{% if user.RUN_ENVIR == "nco" and global.DO_ENSEMBLE %}.${ensmem_name}{% endif %}.prslev.f{lead?fmt=%HHH}.${POST_OUTPUT_DOMAIN_NAME}_a${ACCUM_HH}h.nc'
```

Comment:
While manually running the coverage.hera.intel.nco WE2E tests, I'm encountering the following failure in the `grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2` test, in the run_fcst task:

`FATAL from PE     5: compute_qs: saturation vapor pressure table overflow, nbad=      1`

When I removed the `_default` from `NET_default` on lines 2453, 2454, and 2455, the test once again passes.

If you would like to run the Hera Intel coverage WE2E tests, please set the following environmental variables:
```
setenv SRW_PLATFORM "hera"
setenv SRW_COMPILER "intel"
setenv WORKSPACE "/path/to/your/ufs-srweather-app"
setenv SRW_WE2E_COMPREHENSIVE_TESTS "False"
setenv SRW_PROJECT "enter_your_Hera_allocation_account_here"
```
With the above variables set, you should be able to go into `.cicd/scripts`, then run `./srw_build.sh` to build the SRW App, then `./srw_test.sh` to run the Hera Intel coverage tests.

Alternatively, you should be able to launch the tests without the environment variables, but instead use the following command from `tests/WE2E`:

`./setup_WE2E_tests.sh hera Hera_allocation_account intel coverage --expt_basedir=/path/to/where/you/want/your/expt_dirs --ops_root=/path/to/where/you/want/your/nco_dirs`

where `Hera_allocation_account` in the above command line is your account to run jobs on Hera.

Reply:
@MichaelLueken That's strange because those 3 variables are used only in verification as far as I know (I created them), and the test `grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2` doesn't run any vx tasks.

I will run this test and see what happens.  I normally run the WE2E tests using the `run_WE2E_tests.py` script.  Should that method give the same result as using the `setup_WE2E_tests.sh` (i.e. using your method)?

Reply:
@MichaelLueken Thanks for the explanation.  Since I'm more familiar with `run_WE2E_tests.py`, I first wanted to make sure the test `grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2` runs successfully using this script.  So:
1) I ran this test (in community mode) using `run_WE2E_tests.py`.  The test completed successfully.  You can see the output at:
    ```
    /scratch2/BMC/det/Gerard.Ketefian/UFS_CAM/TEST_SRW_phys_tendencies/expt_dirs/NET_02/grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2
    ```
2) To repeat the same test except in NCO mode, I created a new test in 
    ```
    ufs-srweather-app/tests/WE2E/test_configs/grids_extrn_mdls_suites_nco
    ```
    named `nco_grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2`.  The only difference between this and the original test is that it has `RUN_ENVIR` set to "nco", i.e. in the config file, we have
    <pre>
    user:
      RUN_ENVIR: <b>nco</b></pre>
    I ran this again using `run_WE2E_tests.py`.  This test completed successfully.  You can see the output at:
    ```
    /scratch2/BMC/det/Gerard.Ketefian/UFS_CAM/TEST_SRW_phys_tendencies/expt_dirs/NET_02/nco_grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2
    ```
    and at
    ```
    /scratch2/BMC/det/Gerard.Ketefian/UFS_CAM/TEST_SRW_phys_tendencies/expt_dirs/nco_dirs.works01
    ```
This makes me think the problem is in the way `setup_WE2E_tests.sh` runs the test and/or due to something that was changed by PR #847 since my clone is one PR behind that.  I'll try it with `setup_WE2E_tests.sh`.  If that works, I'll update my hash to PR #847 and retry.

Reply:
@MichaelLueken I tried the way you suggested of rebuilding using `.cicd/scripts/srw_build.sh`, then running `.cicd/scripts/srw_test.sh`.  To make things simpler, I only included `grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2` in `coverage.hera.intel.nco`.  The test succeeded.  You can find the experiment directory at
```
/scratch2/BMC/det/Gerard.Ketefian/UFS_CAM/TEST_SRW_phys_tendencies/ufs-srweather-app/expt_dirs/grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2
```
and the `nco_dirs` directory at
```
/scratch2/BMC/det/Gerard.Ketefian/UFS_CAM/TEST_SRW_phys_tendencies/ufs-srweather-app/nco_dirs
```
Ok, last thing to try is to update my hash (i.e. merge latest `develop` into my branch).

Reply:
@MichaelLueken As a final test, I merged the latest develop (with PR #847) into my branch and reran `srw_test.sh` as above (I didn't rebuild since none of the external repo hashes changed).  The test succeeded.  You can find the experiment directory at
```
/scratch2/BMC/det/Gerard.Ketefian/UFS_CAM/TEST_SRW_phys_tendencies/ufs-srweather-app/expt_dirs/grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2.hash_6e5b4275
```
and the nco_dirs directory at
```
/scratch2/BMC/det/Gerard.Ketefian/UFS_CAM/TEST_SRW_phys_tendencies/ufs-srweather-app/nco_dirs.hash_6e5b4275
```
So I don't know why your tests failed but mine work!